gcp_variant_transforms/beam_io/vcf_header_io.py

```diff
@@ -331,20 +315,17 @@ def __init__(
     self._source = VcfHeaderSource(
         file_pattern,
         compression_type,
-        validate=validate,
-        vcf_parser=vcf_parser)
+        validate=validate)
 
   def expand(self, pvalue):
     return pvalue.pipeline | Read(self._source)
 
 
-def _create_vcf_header_source(
+def CreateVcfHeaderSource(
```

Comment:
Why did you rename this method? This new name does not follow our naming convention (ie its name looks like a class name).

Reply:
Ummm, pylint forced me to do it :( . Are our naming conventions different from pylint ones? That doesn't seem right...

> Function name "create_vcf_header_source" doesn't conform to u'^(?:(?P<camel_case>_?[A-Z][a-zA-Z0-9]*)|(?P<snake_case>_?[a-z][a-z0-9_]*))$' pattern